cmd/snowflake/root.go

```diff
@@ -23,15 +24,16 @@ import (
 	"go.uber.org/zap"
 )
 
-func genSinkURI(s3StoragePath string, flushInterval time.Duration, fileSize int64) (*url.URL, error) {
-	sinkUri, err := url.Parse(s3StoragePath)
+func genSinkURI(storagePath string, flushInterval time.Duration, fileSize int64) (*url.URL, error) {
+	sinkUri, err := url.Parse(storagePath)
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
 	values := sinkUri.Query()
 	values.Add("flush-interval", flushInterval.String())
 	values.Add("file-size", fmt.Sprint(fileSize))
 	values.Add("protocol", "csv")
+	// TODO(lcui2)
```

Comment:
more explicit? What do we need to do?

Reply:
1. I wonder if we can remove the credValue in NewSnowflakeCmd and instead use a single field that can accommodate both the S3 ak&sk and GCS credential file.
2. Can we parse the storagePath at line 227 where we try to get credValue. By doing so, perheps we can avoid some redundant operations.

Reply:
good idea, you can make a try.